app.py

```diff
@@ -185,5 +196,44 @@ def behind():
         return render_template('behind-the-scene.html', username=g.user['client_name'])
 
 
+@app.route('/activity-log/<int:contact_id>', methods=["GET"])
```

Comment:
You haven't read a thing about REST, have you?

Reply:
I usually read about REST everywhere! But the  more I read, the more confused I get :(

Reply:
Are you confused about the naming pattern recommended by REST?

Reply:
Oh the route was reveresed in editing. Sorry for that